DecipherNow/gm-fabric-dashboard
# Fix: PropType errors

Resolves #1214 

## Proposed Changes

  - Added missing or erroneous proptypes
  
## How to Test

  - Click around the app with the console open and confirm there are no more proptype errors
